### ISSUE TITLE
Semantic Segmentation Inference is not working after Finishing Model Training

### ISSUE THREAD

**Issue by mrasyid** [NVIDIA/DIGITS#1425]
Hello,

I was basically following this semantic segmentation tutorial:
https://github.com/NVIDIA/DIGITS/tree/master/examples/semantic-segmentation
except for this step:
"in Pretrained model(s) specify the path to the pre-trained FCN-Alexnet" (I kept the path empty, the reason is I could not (yet) use the pre-trained model after using net_surgery.py script due to parameter mismatch, but the training process is still able to run, so I thought this is not a problem, since without a pre-trained model, I thought the model training is conducted from the scratch, CMIIW).

And this is the main concern of this thread:
**When I do the Inference, the output image is exactly the same as the input image, meaning there is no segmentation result produced.**

If there is any clue on why this is happening and how to fix this, I really appreciate it.
The image below is how it looks like after model training process. You will notice that the train loss graph is quite strange compared to the tutorial example.

![trained_model_segmentation](https://cloud.githubusercontent.com/assets/1209717/22410488/4b115d86-e6dc-11e6-9530-ff12041b1260.png)

Thank you in advance for any hint and solution for this problem.

**Comment by mrasyid** [comment]
The .caffemodel used above is apparently wrong. Instead of using fcn-alexnet, I was using blvc-alexnet. 
So the solution is simply using the model downloaded and processed after network surgery by the net-surgery.py script.

**Comment by montekristo1946** [comment]
Sorry for my question.
Why not learn without pretrain models?

**Comment by ucesfpa** [comment]
Training from scratch is possible. It is just a matter of number of training data. How many images did you use for training?
